routes.nix

```diff
@@ -25,6 +25,7 @@
     cAdvisor.httpPort = 7080;
     caddy.httpPort = 2019;
     grafana.httpPort = 3000;
+    applicationSlug = "dash"; # todo : correct application slug, can be taken from logout URL on provider overview
```

Comment:
Good thinking. I'd just "hardcode" the slug (which is `grafana`) into the logout url for now.